lib/Text/Regex/TDFA/Pattern.hs

```diff
@@ -201,7 +201,7 @@ decodeCharacterClass (PatternSetCharacterClass s) =
     "digit"  -> ['0'..'9']
     "punct"  -> ['\33'..'\47']++['\58'..'\64']++['\91'..'\96']++['\123'..'\126']
     "alpha"  -> ['A'..'Z']++['a'..'z']
-    "graph"  -> ['\41'..'\126']
+    "graph"  -> ['\33'..'\126']
```

Comment:
This _could_ have been wrong by accident, since octal 41 is decimal 33.

In general, it would be good to have some tests.